examples/c/alexnet.c

```diff
@@ -56,16 +56,16 @@ s_model_t *alexnet(shape_t *image_shape, uint32_t arity)
             NULL,                                     //
         },
         x);
-    free_shape_ctx(sc);
-    free_s_layer(c1);
-    free_s_layer(c2);
-    free_s_layer(c3_c4);
-    free_s_layer(c5);
-    free_s_layer(f4096);
-    free_s_layer(f_out);
-    free_s_layer(pool);
-    free_s_layer(relu);
-    free_s_layer(out);
+    del_shape_ctx(sc);
```

Comment:
deleting the layers is confusing to me. Actually we are release layer resources; but the layers, in logic, still exist in the model. The users shall still be able to query the model regarding layer statistics just like what we are doing in TL later.

Reply:
Currently we don't have any logical layers saved in model, the layers are just macros or syntax sugars for constructing the model, they are useless once the model is constructed.